app/content/react/library-reference/widgets.md

```diff
@@ -159,7 +159,7 @@ Renders a `<FormulaWidget />` component, binded to a source at redux. The widget
 | props.id             | <code>string</code>        |         | ID for the widget instance.                                                                                                         |
 | props.title          | <code>string</code>        |         | Title to show in the widget header.                                                                                                 |
 | props.dataSource     | <code>string</code>        |         | ID of the data source to get the data from.                                                                                         |
-| props.column         | <code>string | Array<string></code>        |         | Name of the data source's column(s) to get the data from. If multiples are provided, they will be merged into a single one using joinOperation property. |
+| props.column         | <code>string | Array<string></code>        |         | Name of the data source's column to operate with. If not defined, same as column. If multiple columns are provided, they will be combined using the operation specified with the `joinOperation` property. |
```

Comment:
The FormulaWidget does not have an `operationColumn` prop, so it makes no sense to add "If not defined, same as column".